ksmt-core/src/main/kotlin/io/ksmt/KContext.kt

```diff
@@ -1334,9 +1650,28 @@ open class KContext(
     // integer
     private val intModCache = mkAstInterner<KModIntExpr>()
 
+    /**
+     * Create an Int mod expression.
+     *
+     * The result value is a number `r` such that
+     * [lhs] = `r` + ([rhs] * (div [lhs] [rhs]))
+     * where `div` is an Int division that works according to the [mkArithDiv] rules.
+     * The result value is always positive or zero.
+     *
+     * For example:
+     *   `47 mod 13 = 8`
+     *   `47 mod -13 = 8`
+     *   `-47 mod 13 = 5`
+     *   `-47 mod -13 = 5`
```

Comment:
![image](https://github.com/UnitTestBot/ksmt/assets/31047452/e47afe82-e390-4f78-99c8-f06e99dd8024)

Reply:
Fixed in #115